### PR TITLE
(chore) - bundling

### PR DESCRIPTION
**Pull request by JoviDeCroock**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,16 +25,30 @@
     "exchange"
   ],
   "author": "",
-  "license": "ISC",
+  "license": "MIT",
```
Comment (JoviDeCroock) [inline package.json]: Note that I changed the license to be inline with other urql repo's

**Comment by andyrichardson** [comment]
@JoviDeCroock thanks for this! Looks good.

Is there an advantage to using the typescript plugin provided for rollup as opposed to babel? I've only ever used the babel plugin.

**Comment by JoviDeCroock** [comment]
Babel seems to compile when there are type errors, I don't really like that :D in libraries I always use the plugin.

**Comment by kitten** [comment]
@andyrichardson @jovidecroock we've probably also reached a point where we may want to create a "preset" package for this rollup config 😅

It's currently the same between all urql packages, which is awesome ✨, because some time has gone into it to make sure we get the most compact output! However, I dread the day when we have to manually upgrade it across all repos 😅

**Comment by JoviDeCroock** [comment]
@kitten I'll make work of this tomorrow if you like

**Pull request merged**